common.mk

```diff
@@ -113,7 +116,7 @@ optee-os-common:
 
 OPTEE_OS_CLEAN_COMMON_FLAGS ?= $(OPTEE_OS_COMMON_FLAGS)
 
-optee-os-clean-common:
+optee-os-clean-common: xtest-clean
```

Comment:
OK, but this will not prevent `xtest-clean` from failing the second time anyway, so `optee_test/Makefile` should also be fixed.